controller/src/main/java/org/jboss/as/controller/remote/TransactionalProtocolOperationHandler.java

```diff
@@ -75,6 +88,7 @@
  * @author <a href="mailto:darran.lofthouse@jboss.com">Darran Lofthouse</a>
  */
 public class TransactionalProtocolOperationHandler implements ManagementRequestHandlerFactory {
+    private static final Set<String> PREPARED_RESPONSE_OPERATIONS = new HashSet<>(List.of(RELOAD));
```

Comment:
This should be an immutable set.

Reply:
Done

Reply:
@bstansberry Could you take another look?